demos/loan-platform/README.md

```diff
@@ -0,0 +1,173 @@
+# Loan Platform Use case
+
+# Purpose
+
+This demo is meant to mimic a real world loan platform scenario where multiple financial institutions have to share their data across organizations and make changes to loans. All names are to keep things as realistic as possible.
+
+# Pre-Requisite
```

Comment:
What AWS keypair or identity must be obtained prior to the Terraform portion of the setup script?  I may have missed the location where those are called out but I didn't see that in the write.  Am I missing something that grants the caller of the script access to "their" AWS dev environment?

Reply:
Great point out.  A better written terraform script would have AWS_SECRETS and so in a file. But for our environment we do swap between accounts. So the way I failed to put it in is that before running `terraform apply`, we need to have our AWS CLI setup and working properly.

Reply:
Updated below in README.md to address this:


Have your aws credentials in your environment variable. This is needed for the terraform script.
```
export AWS_ACCESS_KEY_ID=<your-aws-access-token>
export AWS_SECRET_ACCESS_KEY=<your-aws-accee-token-secret>
```